lib/chef/tidy_common.rb

```diff
@@ -149,6 +149,23 @@ def write_new_file(contents, path, backup = true)
       end
     end
 
+    # Read a json file and return a hash of parsed content with optional symbolized keys
+    #
+    # @param [String] path to file
+    # @param [Bool] symbolize keys, defaults to false
+    #
+    # @return [Hash]
+    #
+    # @example
+    # json_file_to_hash('/path/to/file.json', true) => { foo: "bar" }
+    #
+    def json_file_to_hash(file_path, symbolize = false)
+      FFI_Yajl::Parser.parse(File.read(file_path), symbolize_names: symbolize)
+    rescue Errno::ENOENT, Errno::EACCES, FFI_Yajl::ParseError => e
+      puts "ERROR: unable to parse file: '#{file_path}' #{e}"
```

Comment:
This should probably just be a raise to avoid the need to also exit.

Reply:
Great idea. Addressed in https://github.com/chef/knife-tidy/pull/121/commits/13a46b3cbd62b7acf633f5654ab3379cded8bde2